src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -85,6 +85,15 @@
 
                 </Grid>
 
+                <Border x:Name="tfvcInfo" BorderThickness="2" BorderBrush="Red" Margin="15,20,15,0" >
+                    <TextBlock TextWrapping="WrapWithOverflow" Padding="10">
+                        <Run>NB: If you are using Tfvc source control, please read the additional information 
+                        </Run>
+                        <Hyperlink NavigateUri="http://any" RequestNavigate="OnNavigateToTfvcHelp"> here</Hyperlink>
+                            <Run> before starting.</Run>
+                    </TextBlock>
+                </Border>
```

Comment:
This is the other change I discussed with Marco: extra text for Tfvc users.

Currently this is hard-coded. I'll add a ticket to make it dynamic, but we might not have time to do it.